widgets/BigQueryTable-batchsink.json

```diff
@@ -7,6 +7,11 @@
     {
       "label" : "Service Account and Temporary Google Storage",
       "properties" : [
+        {
+          "widget-type": "textbox",
+          "label": "Reference Name",
+          "name": "referenceName"
```

Comment:
there should be a placeholder in the widget attributes that is consistent with what the other plugins use.